app/test_split_integer.py

```diff
@@ -2,20 +2,28 @@
 
 
 def test_sum_of_the_parts_should_be_equal_to_value() -> None:
-    pass
+    assert sum(split_integer(17, 5)) == 17
 
 
 def test_should_split_into_equal_parts_when_value_divisible_by_parts() -> None:
-    pass
+    assert len(set(split_integer(10, 5))) == 1
 
 
 def test_should_return_part_equals_to_value_when_split_into_one_part() -> None:
-    pass
+    assert split_integer(10, 1) == [10]
 
 
 def test_parts_should_be_sorted_when_they_are_not_equal() -> None:
-    pass
+    value, number_of_parts = 16, 5
+
+    expected_res = split_integer(value, number_of_parts)
+
+    assert expected_res == sorted(expected_res)
```

Comment:
there is no need to create additional variables. you can write everything in one line